.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Tell us about a problem you are experiencing
+
+---
+
+/kind bug
+
+**What steps did you take and what happened:**
+[A clear and concise description of what the bug is.]
+
+
+**What did you expect to happen:**
+
+
+**Anything else you would like to add:**
+[Miscellaneous information that will assist in solving the issue.]
+
+
+**Environment:**
+
+- OpenDatahub Version: (please check the operator version)
+- Workbench: (all, data-science, etc)
```

Comment:
It would be nice here the reporter to indicate the notebook-controller version and workbench version

Reply:
Thanks for the suggestion.
Updated it.